common/djangoapps/student/views.py

```diff
@@ -662,8 +662,16 @@ def dashboard(request):
             user, course_org_filter, org_filter_out_set
         )
 
+    if 'notlive' in request.GET:
+        redirect_message = _("The course you're looking for doesn't start until {0}.").format(
```

Comment:
@peter-fogg My only recommendation is not to use contractions, even if it seems a little less friendly: "The course you are looking for does not start until..."
This will be a nice improvement to the student experience!